README.md

```diff
@@ -225,8 +225,8 @@ end
 Route patterns may have optional parameters:
 
 ``` ruby
-get '/posts.?:format?' do
-  # matches "GET /posts" and any extension "GET /posts.json", "GET /posts.xml" etc.
+get '/post/:format?' do
+  # matches "GET /posts" and any extension "GET /posts/json", "GET /posts/xml" etc
```

Comment:
Ah, right. Damn, I missed that.

OK, so if we do `get '/posts/:year?`, then the comment would have to say that it `matches "GET /posts/", which sucks as a route, since people typing things in would tend to leave off the trailing slash.